### PR TITLE
Adding jacoco coverage report

### PR DESCRIPTION
**Pull request by abhijith-prabhakar**

### CODE REVIEW

**Review by fabiocarvalho777 — CHANGES_REQUESTED** [review]
Please add a hashtag followed by issue number at the end of all commit and PR names.

File: build.gradle [hunk 1]
```diff
@@ -54,6 +55,18 @@ subprojects {
         }
     }
 
+    test {
```
Comment (fabiocarvalho777) [inline build.gradle:58]: Just double checking, did you test to see if the report is generated properly per module?

Also, did you check if an aggregate report is generated at the root? For Codacy to work we will need that (or at least that is how it used to be).

**Review by fabiocarvalho777 — CHANGES_REQUESTED** [review]

**Review by AmandeepSingh97 — COMMENTED** [review]
Are we pushing changes directly to master? I read in contribution doc that we have to put them to develop branch.

**Comment by fabiocarvalho777** [comment]
> Are we pushing changes directly to master? I read in contribution doc that we have to put them to develop branch.

@amandeepsingh97 You are totally right Aman!! Thanks for catching that!!

@abhijith-prabhakar please, can you close this PR and create a new one out of develop branch to develop branch? Thanks.

**Pull request closed without merging**